apps/website/src/app/globals.css

```diff
@@ -1,5 +1,3 @@
-@tailwind base;
```

Comment:
?

Reply:
I'm a bit confused why we removed `@tailwind components` and at the time of the comment `@tailwind base` was removed.